tests/phpunit/unit/api/utilities/beansPathToUrl.php

```diff
@@ -21,13 +21,37 @@
  */
 class Tests_BeansPathToUrl extends Test_Case {
 
+	/**
+	 * Relative path to the Beans directory starting from `wp-content`.
+	 *
+	 * @var string
+	 */
+	protected static $beans_relative_path;
```

Comment:
Why making this static?

PS: this comment applies to `$beans_tests_relative_path` and `$abspath` in the `Tests_BeansUrlToPath`?

Reply:
I typically use `static` on properties that don't change, though really these should be constants.  Regardless, it's a carry-over from working on other projects.  I've removed it and switched to protected properties.